types/protocol.d.ts

```diff
@@ -99,33 +19,31 @@ export namespace Protocol {
              * Message source.
              */
             source: ('xml' | 'javascript' | 'network' | 'console-api' | 'storage' | 'appcache' | 'rendering' | 'security' | 'other' | 'deprecation' | 'worker');
-
             /**
              * Message severity.
              */
             level: ('log' | 'warning' | 'error' | 'debug' | 'info');
-
             /**
              * Message text.
              */
             text: string;
-
             /**
              * URL of the message origin.
              */
             url?: string;
-
             /**
              * Line number in the resource that generated this message (1-based).
              */
             line?: integer;
-
```

Comment:
Don't the spaces make the file more readable?

Reply:
> Don't the spaces make the file more readable?

It was way too much whitespace in the mappings file and removing it from interfaces in the protocol file seemed still readable since most have a description comment block. And I didn't want to pipe in a boolean for extra lines or not :)

Can re-add spacing if others agree it's better with it.

Reply:
no strong feelings but i'm fine without the spaces. it's dense but the comments already give things plenty of room.